apps/coscrad-frontend/src/components/Tags/TagIndex.spec.tsx

```diff
@@ -0,0 +1,50 @@
+import { ITag } from '@coscrad/api-interfaces';
+import { screen, waitFor } from '@testing-library/react';
+import { MemoryRouter } from 'react-router-dom';
+import { getConfig } from '../../config';
+import { buildGetHandlers } from '../../utils/test-utils/buildGetHandlers';
+import { renderWithProviders } from '../../utils/test-utils/renderWithProviders';
+import { setupTestServer } from '../../utils/test-utils/setupTestServer';
+import { TagIndexContainer } from './TagIndex.container';
+
+const dummyTags: ITag[] = [
```

Comment:
We may want a helper to seed dummy tags that is shared, but right now, it would only be used twice. We may put some thought to somehow sharing our test data from the backend statically with the frontend.

Reply:
[Here](https://www.pivotaltracker.com/story/show/183619012) is a story for that.